bliss/models/decoder.py

```diff
@@ -377,67 +374,63 @@ def _render_one_source(self, locs: Tensor, source: Tensor):
         grid_loc = local_grid - locs_swapped
         return F.grid_sample(source, grid_loc, align_corners=True)
 
-    def _expand_source(self, source: Tensor):
-        """Pad the source with zeros so that it is size ptile_slen."""
-        assert len(source.shape) == 3
 
-        slen = self.ptile_slen + ((self.ptile_slen % 2) == 0) * 1
-        assert len(source.shape) == 3
+def fit_source_to_ptile(source: Tensor, ptile_slen: int):
```

Comment:
since `fit_source_to_ptile` is not a method of the tiler anymore, you can consider renaming `fit_source_to_slen` since it doesn't necessarily has to be a padded tile we are fitting it to. It's a more general function